components/support/ktx/src/test/java/mozilla/components/support/ktx/kotlin/StringTest.kt

```diff
@@ -34,10 +35,17 @@ class StringTest {
         assertTrue("https://mozilla.org".isUrl())
         assertTrue("file://somefile.txt".isUrl())
         assertTrue("http://mozilla".isUrl())
+        assertTrue("http://192.168.255.255".isUrl())
 
         assertFalse("mozilla".isUrl())
         assertFalse("mozilla android".isUrl())
         assertFalse(" mozilla android ".isUrl())
+        assertFalse("Tweet:".isUrl())
+        assertFalse("inurl:mozilla.org advanced search".isUrl())
+
+        val extraText = "Check out @asa’s Tweet: https://twitter.com/asa/status/123456789?s=09"
+        val url = extraText.split(" ").find { it.isUrl() }
+        assertNotEquals("Tweet:", url)
```

Comment:
Oh, I didn't know that `IntentProcessor` uses `isUrl()` and `split()`.

Some time ago we ported `WebURLFinder` over from Fennec:
https://github.com/mozilla-mobile/android-components/blob/master/components/support/utils/src/main/java/mozilla/components/support/utils/WebURLFinder.kt

Maybe `WebURLFinder` is better suited for finding an URL in the shared text?